Security/Http/Authenticator/RefreshTokenAuthenticator.php

```diff
@@ -69,14 +73,26 @@ public function __construct(
         $this->successHandler = $successHandler;
         $this->failureHandler = $failureHandler;
         $this->options = array_merge([
+            'check_path' => null, // @todo in 2.0, change the default to `/login_check`
```

Comment:
> todo in 2.0, change the default to `/login_check`

This refers to the `check_path` configuration option on the `refresh_jwt` authenticator, right? Why default to `/login_check`, shouldn't it be something related to refresh rather than login?

Reply:
I just took the same path Symfony uses in all of the authenticators for their defaults, this could be anything really.